examples/generic_connector_configs/README.md

```diff
@@ -5,6 +5,7 @@
   * [Docker Registry API](#docker-registry-api)
   * [Elasticsearch API](#elasticsearch-api)
   * [GitHub API](#github-api)
+  * [Facebook API](#facebook-api)
```

Comment:
Should we call this the `Instagram API?`

Reply:
I thought about calling it that, but I double checked and this connector will work with all of the Facebook API's listed [here](https://developers.facebook.com/docs/). I made sure to point out in the description that it could be adjusted for all of them, but I was focussing on Instagram for the example. I think it keeping it as Facebook, will help people recognize that without having to dig too deep